devin_lifeguard.yaml

```diff
@@ -51,8 +52,8 @@ rules:
       variables to minimize stack usage within the function.
   - name: prefer-constant-time
     trigger: >-
-      Implement algorithms in constant time to prevent timing attacks and ensure
-      security.
+      For private key based asymmetric algorithms implement in constant time
+      to prevent timing attacks and ensure security.
```

Comment:
We need to be CT when handling any secrets, sym or asym, and even when just handling them with coding converters (base64 etc.), whenever an entity not already in possession of the secret is in a position to time how long the handling takes.  And in practice we try to always use CT implementations on private/secret key data, since it's too tricky to try to track whether it's strictly necessary.